### DISCUSSION TITLE
Studio-Elf Templates MudBlazor 7.14.0 released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4772] [Show and tell]
Studio-Elf Templates MudBlazor 7.14.0 released
Available in the Marketplace
![image](https://github.com/user-attachments/assets/c2f6af12-84ec-46f1-bead-51db174e6460)

**Comment by lynnmiller24** [comment]
@leigh-pointer , I am fairly new to Oqtane (I have successfully downloaded and configured a new instance of Oqtane and have been successful creating my first custom modules using the Default Module Template.  I am trying to create a new custom module using Module Template MudBlazor 7.6.  I installed Studio-Elf Templates MudBlazor v 7.14.0.  After building my new mudblazor module project and installing it on a page, I get a console error when accessing the page.  I will show screen shots.  Test1 and Test3 were created with the Default Module Template and they work.  Test2 was created with Module Template MudBlazor 7.6 and on that page I am getting the console error.  What am I doing wrong?
![Screenshot 2](https://github.com/user-attachments/assets/51463f40-ee71-4b8e-a243-f03af68661ca)
![Screenshot 3](https://github.com/user-attachments/assets/1afa7647-d022-449f-91f4-dac1fc35f45b)
![Screenshot 4](https://github.com/user-attachments/assets/4eedb3b7-bf95-4716-8295-d86ff97bc248)
![Screenshot 1](https://github.com/user-attachments/assets/c1ee9b86-9158-41b4-9aac-855cb0f497cc)

**Comment by leigh-pointer** [comment]
@lynnmiller24 can you upload the Solution to GitHub so I can investigate?

**Reply by lynnmiller24** [reply]
@leigh-pointer , I uploaded the Test2 custom module solution to Github and added you as an collaborator.  Did you get the invite?

**Reply by leigh-pointer** [reply]
@lynnmiller24 yes i have seen and accepted the invite.  I have a few things to wrap up then I will look into it.

**Reply by lynnmiller24** [reply]
@leigh-pointer No rush.  Thank you.

**Reply by leigh-pointer** [reply]
I have looked at the code, compiled it and it executes ok with no errors.
!! make sure you are reference the latest version of MudBlazor.  There currently is no automatic upgrade path you will need to replace anything created with 7.6 with 7.14
![image](https://github.com/user-attachments/assets/84ea26a6-7baa-4769-a350-36a876700fdd)

**Reply by lynnmiller24** [reply]
I am referencing MudBlazor 7.14.  I still get the same error.  I am running Qqtane 5.2.4.  I will keep digging.  Thank you for testing it out on your installation.

**Reply by leigh-pointer** [reply]
something you can try! navigate to your Oqtane.Server wwwroot folder and clear out what is in _content\MudBlazor folder  This should step should be in the debug.cmd file personally.

**Reply by lynnmiller24** [reply]
Last night, I ended up blowing away my Oqtane instance (code/solution and database) and all my test module code/solutions and started over.  I installed Oqtane and ran the Oqtane install wizard.  I installed Studio-Elf Templates MudBlazor v 7.14.0.  I created a custom module via the MudBlazor template.  I compiled the custom module solution.  Ran Oqtane and installed the custom module on a page.  And it worked!  I created two more custom modules (one using the standard Oqtane template and another one with the MudBlazor template) and those work too.  I don't know what I was doing wrong the first time.  Thank you for your help and reaching back out with another suggestion.

**Reply by leigh-pointer** [reply]
Pleased that all is working out! Always happy to help

**Reply by lynnmiller24** [reply]
I would like to play around with your DynamicForm (Lite).  Do you have any getting started docs?

**Reply by leigh-pointer** [reply]
> I would like to play around with your DynamicForm (Lite).  Do you have any getting started docs?
@lynnmiller24 
Log on to Studio-elf.net and navigatie the DynamicForm sections.
